submitit/slurm/slurm.py

```diff
@@ -398,7 +398,9 @@ def _make_sbatch_string(
     gpus_per_node: tp.Optional[int] = None,
     gpus_per_task: tp.Optional[int] = None,
     setup: tp.Optional[tp.List[str]] = None,
-    mem: str = "64GB",
+    mem: tp.Optional[str] = None,
```

Comment:
You can drop the "tp.Optional", it's automatically inferred when right side is 'None'

Reply:
The other lines which have rhs as None still have the `tp.Optional` are you sure you want me to get rid of it? (it would make the style inconsistent). Alternatively I can get rid of it for all the lines if you like.

Reply:
we chose to get rid of the tp.Optional in the rest of the codebase, but I only did this incrementally, so there are still a number of those like this.

Let's keep it like this here, for consistensy with the rest of the function.